jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -805,6 +805,7 @@ private static void ensureProperties(boolean isInitialization) {
 	jdk.internal.util.StaticProperty.jdkSerialFilter();
 }
 
+/*[IF JAVA_SPEC_VERSION >= 12]*/
```

Comment:
I'd prefer this say `JAVA_SPEC_VERSION > 11` (or if you like, `JAVA_SPEC_VERSION >= 17`) so it makes reference to a LTS version.